python/lsst/skymap/tractInfo.py

```diff
@@ -157,11 +157,17 @@ def findPatch(self, coord):
         @param[in] coord: sky coordinate (afwCoord.Coord)
         @return PatchInfo of patch whose inner bbox contains the specified coord
 
-        @raise LookupError if coord is not in tract
+        @raise LookupError if coord is not in tract or we cannot determine the
+            pixel coordinate (which likely means the coord is off the tract).
 
         @note This routine will be more efficient if coord is ICRS.
         """
-        pixelInd = afwGeom.Point2I(self.getWcs().skyToPixel(coord.toIcrs()))
+        try:
+            pixel = self.getWcs().skyToPixel(coord.toIcrs())
+        except (lsst.pex.exceptions.DomainError, lsst.pex.exceptions.RuntimeError):
```

Comment:
If you want to safely reject `coord` that are not instances of `Coord` it might be safer to do this:

    icrsCoord = coord.toIcrs()
    try:
        pixel = self.getWcs().skyToPixel(icrsCoord)
    ...